README.adoc

```diff
@@ -72,9 +72,11 @@ Before you begin, the following additional tools need to be installed:
 supports various container types, but you will use Docker in this guide. For installation 
 instructions, refer to the official https://docs.docker.com/install/[Docker documentation^].
 
-- *OpenShift account:* To access a {kube} cluster, you must sign up for a Red Hat OpenShift online account. To sign up, refer to the 
-https://manage.openshift.com/register/plan[official website^]. Keep in mind that the creation time depends on resource availability 
-and may take some time.
+- *OpenShift account:* To access a {kube} cluster, you must sign up for a Red Hat OpenShift Online account. There are two options, 
+Starter and Pro. You will use the Starter plan, which includes a free trial of their platform with limited resources, perfect for individual 
+experimentation. The Pro plan includes more resources and has a monthly fee. To sign up, refer to the 
```

Comment:
Update:
"You will use the Starter plan, which includes a free trial of their platform with limited resources, perfect for individual 
experimentation."
To
"Use the Starter plan, which includes a free trial of the OpenShift platform with limited resources, making it perfect for individual experimentation."